client/src/app/containers/Game/CountAction.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import SendIcon from '@material-ui/icons/Send';
+
+import { ActionInputProps } from './types';
+import { useGame } from './useGame';
+
+interface Props extends ActionInputProps {
+    isCrib: boolean;
+}
+
+const CountAction: React.FunctionComponent<Props> = ({
+    isBlocking,
+    isCrib,
+}) => {
+    const [points, setPoints] = useState(0);
+    const { submitCountHandAction, submitCountCribAction } = useGame();
+
+    const submitAction = isCrib ? submitCountCribAction : submitCountHandAction;
+    const componentId = isCrib ? 'count-crib-input' : 'count-hand-input';
+
+    return (
+        <FormGroup row>
+            <FormControl>
+                <InputLabel htmlFor={componentId}>
+                    {isCrib ? 'Crib' : 'Hand'} Points
+                </InputLabel>
+                <Input
+                    disabled={!isCrib && !isBlocking}
```

Comment:
:mag: Shouldn't this just be `!isBlocking`?

Reply:
Prolly